packages/design-system/src/components/Dropdown/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -1,66 +1,199 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Dropdown accepts optgroup children 1`] = `
+<ul
+  aria-labelledby="dropdown__label--50"
+  class="ds-c-dropdown__menu ds-c-dropdown__menu--grouped"
+  id="dropdown__menu--51"
+  role="listbox"
+>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item-group"
+    disabled=""
+    id="downshift-16-item-0"
+    role="group"
+  >
+    Group A
+  </li>
+  <li
+    aria-selected="true"
+    class="ds-c-dropdown__item ds-c-dropdown__item--highlighted ds-c-dropdown__item--selected"
+    id="downshift-16-item-1"
+  >
+    Option A-1
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item"
+    id="downshift-16-item-2"
+  >
+    Option A-2
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item"
+    id="downshift-16-item-3"
+  >
+    Option A-3
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item-group"
+    disabled=""
+    id="downshift-16-item-4"
+    role="group"
+  >
+    Group B
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item"
+    id="downshift-16-item-5"
+  >
+    Option B-1
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item"
+    id="downshift-16-item-6"
+  >
+    Option B-2
+  </li>
+  <li
+    aria-selected="false"
+    class="ds-c-dropdown__item"
+    id="downshift-16-item-7"
+  >
+    Option B-3
+  </li>
+</ul>
+`;
+
 exports[`Dropdown dropdown matches snapshot 1`] = `
 <div
-  class=""
+  class="ds-c-dropdown"
 >
   <label
```

Comment:
~Just dawned on me that it seems weird having a `<label>` associated with a `<button>`. Does the `for` on label and `id` on button work?~

Nevermind... Just read the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) and buttons can be associated with labels. I learned something!

> Elements that can be associated with a <label> element include [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button), [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) (except for type="hidden"), [<meter>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter), [<output>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output), [<progress>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress), [<select>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) and [<textarea>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).